deploy/infraagent-configmap.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
 data:
   infraagent.yaml: |
     interfaceType: tap
+    insecure: true
```

Comment:
Our understanding is that the agent and manager will communicate via mtls. So, can we make insecure false by default?

Reply:
I have added this as temporary config so that deployment of Agent still works while mTLS is not fully implemented. We can remove or change this when TLS is fully functional.